docs/home/defend_your_workstation.md

```diff
@@ -0,0 +1,117 @@
+
+## Prerequisites
+
+* [Create a Phylum account](https://app.phylum.io/)
+* Install the [Phylum CLI tool](https://docs.phylum.io/docs/quickstart)
+  * Allow the default ecosystem extension(s) you plan to use
+* [Login to your account with the CLI tool](https://docs.phylum.io/docs/phylum_auth_login)
+
+## Walkthrough
+
+1. Navigate to the directory of the relevant software project
+2. Prefix your typical package manager installation command with `phylum` to leverage the Phylum protected package installation
+
+   ```sh
+   # Example for `npm`
+   phylum npm install my-package  # This will be checked by Phylum!
+   ```
+
+3. (Optional) Set up an alias in your shell to make it transparent
+
+   ```sh
+   # Example for `npm`
+   alias npm="phylum npm"
+   npm install my-package  # This will be checked by Phylum!
+   ```
+
+### Example: Blocked Install
+
+This is an example of using the Phylum `npm` ecosystem extension to install the `nocivo@2.1.11` package. It contains obfuscated code and was marked as malicious by Phylum, causing an analysis failure and blocked install.
+
+```shellsession
+❯ phylum npm install nocivo@2.1.1
+[phylum] Updating lockfile…
+
+up to date, audited 2 packages in 501ms
+
+found 0 vulnerabilities
+[phylum] Lockfile updated successfully.
+
+[phylum] Analyzing packages…
+[phylum] Phylum Supply Chain Risk Analysis - FAILURE
+
+[npm] nocivo@2.1.1
+ [malicious_code] nocivo@2.1.1 contains obfuscated Javascript
+```
+
+### Example: Allowed Install
+
+This is an example of using the Phylum `npm` ecosystem extension to install the popular `lodash` package. In this case, the resolved version of the package was free of vulnerabilities and passed Phylum analysis. The package was allowed to be installed, but with the protection of the sandbox. No restricted resources were accessed by any build scripts and the install succeeded.
+
+```shellsession
+❯ phylum npm install lodash
+[phylum] Updating lockfile…
+
+up to date, audited 2 packages in 483ms
+
+found 0 vulnerabilities
+[phylum] Lockfile updated successfully.
+
+[phylum] Analyzing packages…
+[phylum] Phylum Supply Chain Risk Analysis - SUCCESS
+
+[phylum] Installing without build scripts…
+
+added 1 package, and audited 2 packages in 461ms
+
+found 0 vulnerabilities
+[phylum] Packages installed successfully.
+
+[phylum] Running build scripts inside sandbox…
+
+up to date in 122ms
+[phylum] Packages built successfully.
+```
+
+## Supported Ecosystems
+
+Phylum provides support for a number of common package managers used in the [supported ecosystems](https://docs.phylum.io/docs/analyzing-dependencies). These include `npm`, `yarn`, `pip`, `poetry`, `bundle`. `cargo` and more on the way! To get the current list, check the [official extensions](https://github.com/phylum-dev/cli/tree/main/extensions). The ecosystem extensions there are included by default when installing the Phylum CLI.
```

Comment:
Currently `npm`, `yarn` and `pip` (and also maybe `poestry`?) leverage the sandbox. The `bundle` and `cargo` do not. idk if we want to call that out here or now.

Reply:
If we can get the specifics, I think we can note that caveat.

Reply:
Dang...another instance of me leaving the review up too long and not seeing other comments before I made my own.

Reply:
`poetry` does not currently execute the package install with the sandbox.
Ref: https://github.com/phylum-dev/cli/blob/main/extensions/poetry/main.ts#L70-L73

Reply:
I added a note here

https://github.com/phylum-dev/documentation/blob/e807cfcd89cfcd794ce491b35c0997dc47b9fddc/docs/home/defend_your_workstation.md?plain=1#L101